### PR TITLE
Port from GObject to GLib

### PR DESCRIPTION
**Pull request by aryanharitsh123**
Working(Tested By Hrishi).

### CODE REVIEW

File: sugargame/canvas.py [hunk 1]
```diff
@@ -1,7 +1,6 @@
 import os
 from gi.repository import Gtk
 from gi.repository import GObject
-from gi.repository import GLib
```
Comment (rhl-bthr) [inline sugargame/canvas.py:4]: Why is this import removed ?

File: sugargame/event.py [hunk 2]
```diff
@@ -1,6 +1,7 @@
 import logging
 from gi.repository import Gdk
 from gi.repository import GObject
```
Comment (rhl-bthr) [inline sugargame/event.py:3]: Please remove this import, as suggested in #5 review

**Review by rhl-bthr — CHANGES_REQUESTED** [review]
Also, please also squash your commits into one. The current commit messages are not descriptive of the changes you are making. See our guide to writing commits: https://github.com/sugarlabs/sugar-docs/blob/master/src/contributing.md#making-commits

The PR is also not linked to the issue being solved.

**Comment by rhl-bthr** [comment]
@aryanharitsh123, if someone else has tested your commits, please include them in the `tested-by` section. It makes it easier for other developers to look up.

Also, the activity doesn't work for me due to the missing import. It is strange that it worked for you / Hrishi

**Comment by quozl** [comment]
Thanks.  Good try.  Was it a trap?  :grin:  @aryanharitsh123, also;
* files in the _sugargame_ directory are from the upstream project https://github.com/sugarlabs/sugargame, and we try to keep them synchronised so that patches can be ported between repositories as needed,
* my method would be to grab the commit from upstream project https://github.com/sugarlabs/sugargame/commit/6d83a59c917f1d05eb0d67027aa453276855480d and apply it here; e.g. `git format-patch` in that repository, then `git am file.patch` in this repository, that way the same code continues to be used and the credit for the change suitably recorded,
* if the commit doesn't apply due to conflicts, there may be earlier commits to apply,
* but the above is best done with the command line interface to `git`, and your commit messages suggest you've been using the GitHub editor; seems easier, but has consequences.

**Comment by quozl** [comment]
New pull request #8 by same author without closing this one, so I'll close this one.  Next time just force push your branch, it's much easier for everyone.

**Pull request closed without merging**